Sources/Widgets/Widgets3D/SeedWidget/behavior.js

```diff
@@ -0,0 +1,95 @@
+import macro from 'vtk.js/Sources/macros';
+
+export default function widgetBehavior(publicAPI, model) {
+  const state = model.widgetState;
+  const moveHandle = state.getMoveHandle();
```

Comment:
Just a note for now: this will cause trouble if someone ever decides to call `widget.setWidgetState(...)` after initialization, since now `state` points to the old widget state. (I think widgetState is settable.)

Reply:
Ok, I just removed that declaration and now `model.widgetState` is referred to directly anywhere that `state` was used. Changed in [b4971cd](https://github.com/Kitware/vtk-js/pull/2907/commits/b4971cd263068dcac79f9d613a60f3733421d80c)